utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/wireless.lua

```diff
@@ -88,6 +88,11 @@ function main.write(self, section, value)
     --share internet was not enabled before, set to false now
     uci:set("ffwizard", "settings", "sharenet", "0")
     uci:save("ffwizard")
+    --in case the wizard has been re-ran, ensure ffuplink is disabled
```

Comment:
why are these changes made in the wireless.lua file? As it acts on the "share internet"-settings it should be better placed in "shareInternet.lua".

Reply:
shareInternet.lua is __only__ called when the user clicks on " Am Freifunk-Netz teilnehmen und Internet teilen" https://github.com/freifunk-berlin/firmware-packages/blob/8e2b55e1e4e9b396562cb0305cdecd912a5f7cda/utils/luci-app-ffwizard-berlin/luasrc/view/freifunk/assistent/decide.htm#L37  

If the user clicks "Am Freifunk-Netz teilnehmen" https://github.com/freifunk-berlin/firmware-packages/blob/8e2b55e1e4e9b396562cb0305cdecd912a5f7cda/utils/luci-app-ffwizard-berlin/luasrc/view/freifunk/assistent/decide.htm#L19 then shareInternet.lua is jumped-over and the wizard goes straight to "optionalConfigs".

When the wizard is run for a second time, and if this code were in "shareInternet", and the user changes from sharing to not sharing, then the code would never run and ffuplink would not be disabled.

Sure the code could be moved to https://github.com/freifunk-berlin/firmware-packages/blob/8e2b55e1e4e9b396562cb0305cdecd912a5f7cda/utils/luci-app-ffwizard-berlin/luasrc/model/cbi/freifunk/assistent/optionalConfigs.lua#L10-L13 with an else clause, but that still is not in "shareInternet".  Therefore "shareInternet.lua" is not a better place.